core/cap-0035.md

```diff
@@ -275,7 +469,12 @@ approve individual transactions and prevent the creation of a claimable balance.
 The change does not have an affect on previous assets, accounts, or transaction 
 structure. It should not cause a breaking change in existing implementations. 
 
-The new operations introduced are all reversable meaning that their use if
+Pre-authorized, pre-signed, or pre-planned transactions that create accounts,
+and create claimable balances, using assets that become clawback enabled
+could fail if those transactions did not include the issuer account as a
+claimant.
+
+The new operations introduced are all reversible meaning that their use if

```

Comment:
This isn't quite true anymore.

Reply:
💯 Thanks for catching this inconsistency. 567e109